uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Upgrader/Step/1021.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+trait CRM_HRLeaveAndAbsences_Upgrader_Step_1021 {
+  
+  /**
+   * Create new option group hrleaveandabsences_work_pattern_change_reason
+   * with values As per contract, Change in contractual hours and
+   * Change in contract type.
+   *
+   * @return bool
+   */
+  public function upgrade_1021() {
+    $result = civicrm_api3('OptionGroup', 'get', [
+      'name' => 'hrleaveandabsences_work_pattern_change_reason'
```

Comment:
I missed this before, but you already have a variable with this name, so you can use it here.